app/controllers/sessions_controller.rb

```diff
@@ -4,15 +4,15 @@ def new
 
   def create
     if login(params[:email], params[:password])
-      redirect_back_or_to root_path, notice: 'Добро пожаловать!'
+      redirect_back_or_to root_path, notice: t('flash.log_in')
     else
-      flash.now[:alert] = 'Пароль или E-mail введен неверно!'
+      flash.now[:alert] = t('flash.wrong_email_or_password')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.